scripts/xtensa-build-all.sh

```diff
@@ -435,5 +435,9 @@ do
 done # for platform in ...
 
 # list all the images
-ls -l build_*/*.ri build_*/src/arch/xtensa/rom*.bin || true
-ls -l build_*/sof
+if [[ "x$BUILD_ROM" == "xyes" ]]
```

Comment:
This is inconsistent and confusing because `sof` and `*.ri` files from previous runs are always listed, whereas `rom*.bin` files from previous runs are listed only when `BUILD_ROM` is true in the current run.

To always list all the `rom*.bin` files from all previous runs (TESTED)
```suggestion
romfiles=(build_*/src/arch/xtensa/rom*.bin)
if test -e "${romfiles[0]}"; then
     ls -l "${romfiles[@]}"
fi
```